src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -2107,29 +2107,6 @@ def test_success(self):
         log_items = ActivityLog.objects.for_addons(self.addon)
         assert log_items.filter(action=amo.LOG.ADD_VERSION.id)
 
-    @mock.patch('olympia.devhub.views.sign_file')
-    def test_experiments_are_auto_signed(self, mock_sign_file):
```

Comment:
So the conclusion was that webextension experiments are completely gone, not just replaced with something in the manifest?

Reply:
There is another test, `test_experiments_inside_webext_are_auto_signed` that already tests WebExtension style experiments (manifest.json based). So this test here got removed.